docs/getting-started/guides/how-to-understand-strategies-descriptions.md

```diff
@@ -0,0 +1,111 @@
+### Liquidation
+- When you fail to repay a [loan](#lend--borrow-against) or other agreement which involves [collateral](#collateral) and the collateral is sent to other parties to repay and make them whole, so you can't access the collateral anymore.
+- For example: you add 100 Eth as collateral and borrow 75% of it's value as dollars, but you fail to repay the loan in due time, so your collateral gets *liquidated* which costs you 25% of the initial amount deposited.
+
+### Lend / Borrow Against
+- Lending is when you provide liquidity to someone else for some time and expect them to pay you back in the future.
+- Borrowing is when you take money from a lender.
+- Borrowing *against* is used to specify the collateral used by the borrower. Example: "I borrowed 5000$ against my car, so if I don't repay the loan the bank can take it".
+- In DeFi this operation is always collateralized since there are no resources to charge debt.
+
+# DeFi
+
+### Yield Aggregator / Yield Optimizer
+- DeFi protocols like Yearn that provides a single interface that aggregates/optimizes users deposits in a single place and reallocates them in an optimal way.
+### Supply / Deposit / Stake
+- Might be used interchangeably sometimes, but it often means that tokens are sent to a contract external to a [strategy](#strategy).
+- Supply often means that the money is being [lent](#lend--borrow-against)
+### Harvest / Collect / Claim
+- When a [strategy](#strategy) collects [profits](#yield--profit) or [rewards](#rewards) in a blockchain transaction.
+### Sell / Buy
+- Market buy or sell tokens on a Decentralized Exchange like [Uniswap](#uniswap) or Sushiswap.
+### Rewards
+- Tokens earned by a protocol-incentivized action.
+- For example: if you deposit liquidity into Uniswap you get trading fees, but you also get UNI *rewards* for doing this at Uniswap and not at SushiSwap.
+### Lock
+- Tokens that are locked can't be used until a specific condition is met.
+### Unlock
+- Unlocking refers to meeting a condition and receiving locked tokens.
+### Liquidity Provider
+- Liquidity Provider is anyone that invest tokens in a protocol.
+### Joint Liquidity Provider
+- Joint [Liquidity Provider](#liquidity-provider) is when your tokens are aggregated with another person's tokens in order to provide liquidity together.
+### Liquidity Pool
+- A [Liquidity Pool↗](https://www.coindesk.com/learn/what-are-liquidity-pools/) is a decentralized market for 2 (or more) tokens.
+### Stable Pool
+- A Stable Pool is a Liquidity Pool made only of stablecoins.
+### Impermanent Loss
+- When you lose money for being a liquidity provider to liquidity pools when compared to just holding the 2 (or more) tokens in your wallet, it happens when the market price ratio for the tokens diverges from the value that they had when you provided liquidity, which is why it can also be called Divergence Loss.
```

Comment:
IL only happens when you enter single-sided, so it's always compared to holding the token you deposited. ie you zap with ETH into the ETH/USDC pool (50% / 50%). If ETH moves - up or down -, you compare your holdings in the pool vs the amount of money that you would have if you've hodled ETH.

Reply:
idk how exactly to change this one, is there something wrong with the current paragraph or is this a suggestion for a new one? ty!